### DISCUSSION TITLE
Multiple Data set Barchart question

### DISCUSSION THREAD

**Discussion by ahmadnav** [GSI-CS-CO/chart-fx#488] [Q&A]
Hi  I am wondering if chartfx allows for adding multiple bars for a category, in a category graph. The following is what I get when I try to add 2 data sets to the ErrorDataSetRenderer, the bars overlay each other. Is there any way the blue and red bars would take half the space of a category and appear side by side? Thank you for your help in advance. 

        for (int n = 0; n < sampleSize+1; n++) {
            dataSet.add(n,n);
            dataSet.addDataLabel(n,String.valueOf(n));
            dataSet1.add(n,n*2);
            dataSet1.addDataLabel(n,String.valueOf(n));
        }

![categorychart](https://user-images.githubusercontent.com/38476413/154410160-6dd008b0-42c1-4d1f-be0e-07c6e5696854.png)

**Comment by RalphSteinhagen** [comment]
Hey @ahmadnav thanks for your interest in ChartFX.

What you want to achieve is already implemented. Maybe  have a look at the samples, e.g.  [HistogramRendererTests.java](
https://github.com/GSI-CS-CO/chart-fx/blob/master/chartfx-chart/src/test/java/de/gsi/chart/renderer/spi/HistogramRendererTests.java)

There are various options how and what can be done, e.g.:
![image](https://user-images.githubusercontent.com/46007894/154448628-74ffdede-6ca9-4c40-9222-4dec756bd899.png)
